src/file_operations.js

```diff
@@ -121,10 +121,7 @@ const getMasterJsonFileName = async coursePath => {
   if (await directoryExists(coursePath)) {
     // If the item is indeed a directory, read all files in it
     const contents = await fsPromises.readdir(coursePath)
-    const fileName = contents.find(
-      file =>
-        RegExp("^[0-9a-f]{32}_master.json").test(file) || file === "master.json"
-    )
+    const fileName = contents.find(file => RegExp(".*_parsed.json").test(file))
```

Comment:
Can we end with `$` so that `_parsed.json.zip` doesn't match for example?

Reply:
all set